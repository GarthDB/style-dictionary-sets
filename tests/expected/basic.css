:root {
  --component-size: 12px;
  --component-black-font-weight: 900;
}
