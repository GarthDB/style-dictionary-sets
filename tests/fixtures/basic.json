{
  "component": {
    "size": {
      "sets": {
        "mobile": {
          "value": "15px"
        },
        "desktop": {
          "value": "12px"
        }
      }
    }
  }
}
