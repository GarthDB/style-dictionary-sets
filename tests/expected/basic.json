{
  "component": {
    "size": {
      "sets": {
        "mobile": {
          "value": "15px",
          "filePath": "tests/fixtures/basic.json",
          "isSource": true,
          "original": {
            "value": "15px"
          },
          "name": "component-size-sets-mobile",
          "attributes": {
            "sets": ["mobile"]
          },
          "path": ["component", "size", "sets", "mobile"]
        },
        "desktop": {
          "value": "12px",
          "filePath": "tests/fixtures/basic.json",
          "isSource": true,
          "original": {
            "value": "12px"
          },
          "name": "component-size-sets-desktop",
          "attributes": {
            "sets": ["desktop"]
          },
          "path": ["component", "size", "sets", "desktop"]
        }
      }
    },
    "black-font-weight": {
      "sets": {
        "spectrum": {
          "attributes": {
            "sets": ["spectrum"]
          },
          "filePath": "tests/fixtures/basic.json",
          "isSource": true,
          "original": {
            "value": "black"
          },
          "name": "component-black-font-weight-sets-spectrum",
          "path": ["component", "black-font-weight", "sets", "spectrum"],
          "value": "black"
        }
      }
    }
  }
}
